index.js

```diff
@@ -42,68 +44,70 @@ function extractLinux(line) {
 	// 6 optional user command interpreter
 
 	return {
-		username: cols[0],
-		password: cols[1],
-		uid: Number(cols[2]),
-		gid: Number(cols[3]),
-		fullname: cols[4] && cols[4].split(',')[0],
-		homedir: cols[5],
-		shell: cols[6]
+		userName: columns[0],
```

Comment:
`username` is a word in itself though, so it should be all lowercase.

Reply:
Yeah, you're right, it was in the heat of the moment...

Reply:
Hehe. I've personally gone back and forth on that one. The inconsistency kinda annoys me `username` vs `fullName`/`userData`, etc, but since it's a word, it's better to leave it as is. Same with `filename`, which is also a word, and `filePath`, since there's is no word filepath, but rather file path.